sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/object.scala

```diff
@@ -93,6 +95,72 @@ case class SerializeFromObject(
   override def output: Seq[Attribute] = serializer.map(_.toAttribute)
 }
 
+object SerializeFromObject {
```

Comment:
@cloud-fan Not sure if this is proper location. If not, please let me know. Thanks.